### DISCUSSION TITLE
Mention User pada Grup

### DISCUSSION THREAD

**Discussion by rioastamal** [kirimwa-id/forum#43] [Roadmap]
Dengan adanya fitur ini maka saat melakukan pengiriman ke grup, pengembang dapat melakukan mention dengan menyebut pengguna lain diawali dengan nomor WhatsAppnya Contoh `@6281234567890`.

Tidak ada parameter baru yang harus ditambahkan oleh pengembang ketika menggunakan fungsi ini.

**Comment by rioastamal** [comment]
Fitur telah diimplementasikan di v1.0-RC8. Lihat di #50

**Comment by melodiazz** [comment]
Nice pak @rioastamal, Thanks!

**Comment by melodiazz** [comment]
Hi Pak @rioastamal 
Mungkin saat ini bukan hal yang urgent tapi tidak apa-apa ya saya submit dulu reportnya.

Fitur mention user pada grup sepertinya belum bisa berhasil semenjak versi support MD rilis.

**Reply by rioastamal** [reply]
Terima kasih laporannya.

**Reply by rioastamal** [reply]
@melodiazz Sudah diperbaiki pada v1.0-RC15. Silahkan dicoba.

**Reply by melodiazz** [reply]
nice pak @rioastamal.. super fast perbaikannya hehehe
